src/Components/App/App.scss

```diff
@@ -3,15 +3,15 @@
   margin: 0;
 };
 
-button {
-  border-radius: 10%;
-  color: #f3f5f6;
-  height: 3vh;
-  border: solid 1px #121316;
-  background-color: #121316;
-  width: 20vw;
-  margin: 1%;
-}
+// button {
```

Comment:
Does this eliminate the button styling throughout the app?

Reply:
So yes and no. For some reason this styling on app, was over riding styling on the feeling room buttons.
I did tweak a few buttons, and the styled components should be the same.
I commented it out though so that if we stuble across an issue we can refer back to here and add styling to the COmponents scss file and not app's.